site/docs/resources.mdx

```diff
@@ -22,6 +22,9 @@ import Link from "../src/components/Link";
 
 - <Link href="https://brand.hel.fi/en/" external>Helsinki Visual Identity guidelines</Link>
 - <Link href="https://www.hel.fi/static/liitteet/kanslia/TPR/opas_saavutettavaan_sisaltoon_EN.pdf" external>Helsinki content accessibility guidelines</Link>
+- <Link href="https://saavutettavuusmalli.hel.fi" external> Helsinki Accessibility Model (in Finnish mostly) </Link>  
+- <Link href="https://pelikirja.hel.fi" external> Helsinki Digital Sevice Playbook (Pelikirja) (in Finnish) </Link> 
```

Comment:
Small typo? Helsinki Digital **Sevice** Playbook

Reply:
Thanks, fixed and committed new version.